lib/bosh/workspace/project_deployment.rb

```diff
@@ -63,14 +54,34 @@ def execute_stub(stub_file)
 
     private
 
+    def stubs_dir
+      File.expand_path(File.join(file_dirname, "../stubs"))
+    end
+
+    def stub_file
+      Dir[File.join(stubs_dir, "#{file_basename('.*')}*")].first
```

Comment:
So it is doing `look for file in stubs folder with the same name as a deployment name` but it ignores extensions.